app/javascript/app/components/definition-list-legacy/definition-list-legacy-styles.scss

```diff
@@ -0,0 +1,51 @@
+@import '~styles/layout.scss';
+
+.definitionCompare {
+  @include columns((3, 3, 3, 3));
+  @include xy-gutters(40px, $gutter-position: ('top', 'bottom'), $gutter-type: padding);
+  @include xy-gutters(60px);
+
+  border-bottom: solid 1px rgba(#9090b2, 0.15);
```

Comment:
Can't we use one of the colors that we already have? In case of doubt please ping Fanny.

Reply:
This is a legacy component, it was like that since a beginning so I'm not sure if it's worth changing anything there since we're going to remove it anyway soon, what do you think?

Reply:
Ah, sure I didn't realize it was de legacy component. Let it be